pom.xml

```diff
@@ -101,6 +101,32 @@
         </pluginManagement>
 
         <plugins>
+            <plugin>
```

Comment:
Kindly remove this plugin. I have an alternative.

Reply:
Done